code42.json

```diff
@@ -1065,6 +1065,243 @@
                 "type": "table"
             },
             "versions": "EQ(*)"
+        },
+        {
+            "action": "create user",
+            "identifier": "create_user",
+            "description": "Create a new Code42 user account",
+            "type": "contain",
+            "read_only": false,
+            "parameters": {
+                "org_uid": {
+                    "description":  "Org UID to create user in",
+                    "data_type": "string",
+                    "required": true,
+                    "primary": true,
+                    "contains": [
+                        "org_id"
+                    ],
+                    "default": "",
+                    "order": 0
+                },
+                "username": {
+                    "description":  "Username to create",
+                    "data_type": "string",
+                    "required": true,
+                    "primary": true,
+                    "contains": [
+                        "email"
+                    ],
+                    "default": "",
+                    "order": 1
+                },
+                "password": {
+                    "description":  "Password for newly created user",
+                    "verbose": "Defaults to no password set, which leaves user in 'invited' state. They will need to reset password via email prior to first login.",
+                    "data_type": "password",
+                    "required": false,
+                    "primary": false,
+                    "default": null,
+                    "order": 3
+                },
+                "first_name": {
+                    "description":  "User's first name",
+                    "data_type": "string",
+                    "required": false,
+                    "primary": false,
+                    "default": "",
+                    "order": 4
+                },
+                "last_name": {
+                    "description":  "User's last name",
+                    "data_type": "string",
+                    "required": false,
+                    "primary": false,
+                    "default": "",
+                    "order": 5
+                },
+                "notes": {
+                    "description":  "Notes about user",
+                    "data_type": "string",
+                    "required": false,
+                    "primary": false,
+                    "default": "",
+                    "order": 6
+                }
+
+            },
+            "output": [
+                {
+                    "data_path": "action_result.parameter.username",
+                    "data_type": "string",
+                    "contains": [
+                        "email"
+                    ],
+                    "column_name": "Username",
+                    "column_order": 0
+                },
+                {
+                    "data_path": "action_result.data.userUid",
```

Comment:
I am little concerned we will confuse people by using `userUid` to mean the same as `userId`.

XSOAR does output the `userUid` as `UserId` for these user-actions.. However, XSOAR also does not output anything else for these methods.

Thoughts?

Reply:
I think we have other stuff that already uses `userId` too, so I think sticking with that makes sense if we intend to output it. Do we even have to output it though? All the actions take username as input 🤷🏼‍♂️